internal/v1/handler_compose_image.go

```diff
@@ -830,6 +830,24 @@ func buildCustomizations(cust *Customizations) (*composer.Customizations, error)
 		}
 	}
 
+	// we need to explicitly add 'rhcd' to the enabled services
+	// if openscap and subscription customizations are set, otherwise
+	// the insights-client doesn't register properly
+	if cust.Subscription != nil && cust.Subscription.Insights && cust.Openscap != nil {
+		if res.Services == nil {
```

Comment:
This is a bit ugly. But it's kind of hard to check for all the cases because we're checking `OpenSCAP`, `Services` and `Subscriptions` customizations